src/runtime_src/core/pcie/tools/xbmgmt/cmd_hotplug.cpp

```diff
@@ -0,0 +1,197 @@
+/**
+ * Copyright (C) 2019 Xilinx, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <string>
+#include <iostream>
+#include <fstream>
+#include <climits>
+#include <getopt.h>
+#include <dirent.h>
+#include <string.h>
+
+#include <boost/filesystem.hpp>
+
+#include "xbmgmt.h"
+#include "core/pcie/linux/scan.h"
+
+using namespace boost::filesystem;
```

Comment:
Please do not use "using" construct as it pollutes namespace.

Reply:
Removed.